.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +0,0 @@
-Thank you for filing! Check list:
```

Comment:
Removed this file and the next by mistake. Needs to be restored.

Reply:
Yes, thanks, I will rebase and clean up this PR anyway and will undo the removal then.